data/genesis/demo.toml

```diff
@@ -33,4 +33,11 @@ version = "0.3"
 start_proposing_view = 5
 stop_proposing_view = 15
 
-[upgrade.marketplace]
\ No newline at end of file
+[upgrade.marketplace]
+[upgrade.marketplace.chain_config]
+chain_id = 12345
```

Comment:
can we please keep the chain id and the formatting consistent with the above chain configs

Reply:
a81aa0f44645f47cfb2735851b2674bb0cefcdba